yt_dlp/extractor/nebula.py

```diff
@@ -169,14 +167,25 @@ class NebulaIE(NebulaBaseIE):
                 'channel_id': 'lindsayellis',
                 'uploader': 'Lindsay Ellis',
                 'uploader_id': 'lindsayellis',
+
+                'timestamp': 1533009600,
+                'uploader_url': 'https://nebula.app/lindsayellis',
+                'series': 'Lindsay Ellis',
+                'average_rating': 0,
+                'display_id': 'that-time-disney-remade-beauty-and-the-beast',
+                'channel_url': 'https://nebula.app/lindsayellis',
+                'creator': 'Lindsay Ellis',
+                'duration': 2212,
+                'view_count': int,
+                'thumbnail': 'https://dj423fildxgac.cloudfront.net/d45d597a-8a81-4580-bc8c-69ab4ec6a8f1.jpeg?height=720&',
             },
             'params': {
                 'usenetrc': True,
             },
```

Comment:
Could u remove the `'usenetrc': True`. At the time of the original PR, I didn't know this, but this should be actually be added to `test/local_parameters.json` instead

Reply:
Makes sense!

For the record, this requires
```
{
    "usenetrc": true
}
```
in the file `test/local_parameters.json`. Specifying `username` and `password` directly doesn't work, because it would be supplied to the Zype extractor as well, and that can't take any.